### PR TITLE
Bisphenols

### PR DESCRIPTION
**Pull request by mattjmeier**
Wanted to flag some important changes here...

1) most importantly, there was an edge case (or maybe not edge case...) where the way columns were selected for checking spikes, relevance filtering, and the quantile rule was not getting the correct samples, leading to way higher numbers of genes being filtered.
2) to help troubleshoot things I've been migrating some stuff to plotly - the PCAs (makes it easier to identify outlier samples by name) and gene plots.
3) gene plots now also have a somewhat convoluted (but as far as I can see, one of the only ways in a static HTML format) method for selecting from the top ~1000 genes by fold change to plot in boxplot format, similar to the static gene plots. Mainly useful for dose-response modeling studies, but pretty helpful when people want to see their 'favorite gene' from the study and it isn't in the top 20. Room for improvement but I think it's a good start!

I've attempted to organize the commits in a logical way so you can look at them in the relevant chunks. I'm learning!

### CODE REVIEW

File: Rmd/DESeq2_report.rnaseq.Rmd [hunk 1]
```diff
@@ -918,9 +931,10 @@ for (x in 1:nrow(contrasts)) {	# For all comparisons to be done
 	
 	# Create output tables
 	norm_data <<- counts(dds[rownames(compte)], normalized = TRUE)
+	# DEsamples <<- subset(res, res$pvalue < 0.01) # Only use in cases of desperation
```
Comment (cookkate) [inline Rmd/DESeq2_report.rnaseq.Rmd:934]: do we need this in here?

File: Rmd/DESeq2_report.rnaseq.Rmd [hunk 2]
```diff
@@ -936,24 +950,31 @@ for (x in 1:nrow(contrasts)) {	# For all comparisons to be done
...
+		Check <- median(as.numeric(DECounts[gene, sampleNames_g2])) > quantile(DECounts[gene, sampleNames_g1], 0.75)[[1]]
 		quantilePass <- c(quantilePass, Check)
 			
-		if (sum(quantilePass) > 0) {
+		if (sum(quantilePass) > 0 ) {
 		  Filter[gene, 2] <- 1
-		}	else {
+		# }	else if (Platform == "TempO-Seq") { # This is a hack... TempOSeq data is messy, so we are ignoring this filter altogether.
+		#   Filter[gene, 2] <- 1
+		} else {
```
Comment (cookkate) [inline Rmd/DESeq2_report.rnaseq.Rmd:967]: ditto. Not sure if the commented out lines were code you were using for tests or if you want to keep them in the code

**Review by cookkate — APPROVED** [review]
Looks good! The interactive plots are neat. I commented on a couple comments that I'm not sure you meant to leave in there.

File: Rmd/DESeq2_report.rnaseq.Rmd (hunk 2 above)
Comment (mattjmeier) [reply Rmd/DESeq2_report.rnaseq.Rmd:967]: Good points, they were for testing before I figured out what was going on with the quantile filtering. Will remove the commented things you noted.

**Comment by mattjmeier** [comment]
I think this is done and ready to merge

**Review by cookkate — APPROVED** [review]

**Pull request merged**